specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -273,17 +273,15 @@
             "$ref": "./examples/Rooms_GetParticipants.json"
           }
         }
-      }
-    },
-    "/rooms/{roomId}/participants:add": {
-      "post": {
+      },
+      "patch": {
         "tags": [
           "Participants"
         ],
-        "summary": "Adds participants to a room. If participants already exist, no change occurs.",
-        "operationId": "Rooms_AddParticipants",
+        "summary": "Upsert participants to a room",
+        "operationId": "Rooms_PatchParticipants",
```

Comment:
nit: do you want to call this Rooms_UpdateParticipants (to align with Rooms_UpdateRoom), and also rename other references to "patch" (e.g. "patchParticipantsRequest")? I don't feel too strongly either way, but I see we use the "update" language in e.g. [phonenumbers](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/communication/data-plane/PhoneNumbers/stable/2021-03-07/phonenumbers.json).

Reply:
Hi @mbakalov thanks for your comments.

For Rooms, we have both PUT and PATCH operations so the PATCH for rooms is used for updates. Hence, naming Rooms models with a term "update". That being said, I see "patch" is being used in Rooms too (e.g. "patchRoomRequest"). This nit seems like inconsistent in Rooms that we may need to make change one way or the other. 

On Participants, there is no PUT but PATCH only, hence naming Participant models with a term "patch". We are using "patch" term consistently across participant models.

Are you ok with keeping the name unchanged for participants?

Reply:
Does "Upsert" make more sense for Participants in this case? This is for updating and insert (also deleting) from the participants list.

Reply:
Noted and will ask ARB for advice here. For the time being, keeping as PATCH